Portal/src/Datahub.Infrastructure/Services/WorkspaceVersionService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Datahub.Application.Services;
+using Datahub.Core.Model.Context;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Datahub.Infrastructure.Services
+{
+    public class WorkspaceVersionService(
+        IDbContextFactory<DatahubProjectDBContext> datahubProjectDbFactory,
+        ILogger<WorkspaceCreationService> logger) : IWorkspaceVersionService
+    {
+        public async Task<string> GetLatestVersion()
+        {
+            await using var db = await datahubProjectDbFactory.CreateDbContextAsync();
+            var versionTags = await db.VersionTags
+                                .Select(t => t.Tag)
+                                .ToListAsync();
+         
+            var latest = versionTags
+                 .Select(v => Version.Parse(v))
+                 .OrderByDescending(v => v)
+                 .First();
+
+            return $"v{latest.ToString()}";
+        }
```

Comment:
I would make the changes below. Be careful because in your code `Version.Parse(v)` fails because the "v" in "v5.0.5" needs to be removed:
![image](https://github.com/user-attachments/assets/bece229e-1de3-46aa-ba56-6b4059bcaf55)

Thats why I added a `TrimStart`. You can also fully skip the selects and make this a single query:
![image](https://github.com/user-attachments/assets/f9d6d174-e3ca-42a7-9eda-89202745674d)

```suggestion
        public async Task<string> GetLatestVersion()
        {
            await using var db = await datahubProjectDbFactory.CreateDbContextAsync();
         
            var latest = db.VersionTags
                 .OrderByDescending(t => Version.Parse(t.Tag.TrimStart('v')))
                 .First();

            return latest.Tag;
        }
```